sbin/root-mount

```diff
@@ -149,9 +150,15 @@ if ! lvm vgchange -a y sailfish; then
 	exit 1
 fi
 
-# Check factory reset need.
+# Check factory reset need and read FS resize status.
 if mount -t ext4 $ROOTDEV $1; then
 	factory_reset_if_needed $1
+
+	IS_FS_RESIZED=0
```

Comment:
Maybe introduce the IS_FS_RESIZED with FS_RESIZED variable, a bit more readable.